rmw_zenoh_cpp/src/detail/event.hpp

```diff
@@ -53,6 +53,8 @@ enum rmw_zenoh_event_type_t
 
 rmw_zenoh_event_type_t zenoh_event_from_rmw_event(rmw_event_type_t rmw_event_type);
 
+bool event_check_compatible(rmw_event_type_t rmw_event_type);
```

Comment:
If we're keeping this function, we should update `rmw_event.cpp` to rely on it. Else we can remove this.